tessellation/src/path_stroke.rs

```diff
@@ -746,6 +743,79 @@ impl<'l, Output: 'l + GeometryBuilder<Vertex>> StrokeBuilder<'l, Output> {
 
         (start_vertex, last_vertex)
     }
+
+    fn tessellate_miter_clip_join(
+        &mut self,
+        prev_tangent: Vec2,
+        next_tangent: Vec2,
+        front_side: Side,
+        back_vertex: VertexId,
+        normal: Vec2,
+    ) -> (VertexId, VertexId) {
+        if self.miter_limit_is_exceeded(normal) {
```

Comment:
nit: really a minor thing but I tend to prefer when the complex code is out of the nested blocks, for example:

```rust
if condition {
    // complicated stuff
    return ...
}
return simple_stuff()
``` 

could be implemented like:

```rust
if !condition {
    return simple_stuff()
}
// complicated stuff
return ...
``` 

It helps with keeping the cyclomatic complexity low (what a scary word!) and and in my experience the code looks a tad less intimidating that way.